exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "06585243-2e50-4f9b-8d6f-b79aade6f8f5",
+   "metadata": {},
+   "source": [
+    "# Using the Script-Language Container\n",
+    "A [Script-Language Container](https://github.com/exasol/script-languages-release) for the Exasol database consists of a Linux container with a complete Linux distribution and all required libraries, such as a script client. The script client is responsible for the communication with the  database and for executing the script code. It allows to also include user specific libraries which can then be used from within the UDF's. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39333102-7652-4699-ba0e-d184aeec1753",
+   "metadata": {},
+   "source": [
+    "Before we start we need to configure the script-languages directory and flavor. See [Configure SLC repository](./configure_slc_flavor.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25604520-c997-4d2b-8138-02a64e7231d0",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "### Open Secure Configuration Storage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da57e77a-b41f-45e3-9a17-4b474bba898b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ../utils/access_store_ui.ipynb\n",
+    "display(get_access_store_ui('../'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81fbc9a1-718a-4d93-8753-85b032998c1b",
+   "metadata": {},
+   "source": [
+    "### Instantiate SLCT Manager\n",
+    "\n",
+    "We create an instance of the SlctManager class from the notebook connector.\n",
+    "This class has some utility function which simplifies the use of the `exaslct` API."
```

Comment:
Rather than from a technical perspective, could you please explain what the SlcManager is used for from a user's perspective? E.g. name some typical actions or say "it helps to ....".